docs/src/js/components/ElementsInput.react.js

```diff
@@ -531,6 +532,12 @@ export default class ElementsButton extends React.Component {
                 default: '',
                 description: 'Specifies that the user must fill in a value before submitting a form.',
                 allowedTypes: ''
+            }, {
+                name: 'showSpinners',
+                type: 'bool',
+                default: 'true',
+                description: 'Show or not the spinners for numeric control. This attribute is optional.',
```

Comment:
Nitpick: `'Specifies whether or not to show the spinners for the numeric control.  This attribute is optional.'`